concepts/index.md

```diff
@@ -6,29 +6,38 @@ layout: documentation
 
 # Concepts
 
-Eclipse SmartHome strictly differentiates between the physical view and the functional view on the system.
-While the physical view is required for setup, configuration, troubleshooting, etc., the functional side covers the information that matter to applications, such as user interfaces and automation logic.
+openHAB is built around two distinctly different views.
+The first is the physical view - devices, configurations, connections, etc.
+The second is the function view.
```

Comment:
functional

Reply:
Agree - this will be addressed in my next commit.